src/components/MultiHopTrade/components/SharedConfirm/AssetSummaryStep.tsx

```diff
@@ -76,7 +76,7 @@ export const AssetSummaryStep = ({
       )
     )
       return
-    if (amountCryptoBaseUnit === prevAmountCryptoBaseUnit) return
+    if (bn(amountCryptoBaseUnit).gte(prevAmountCryptoBaseUnit)) return
```

Comment:
I am seeing the modal fire from `ExpandedStepperSteps`, so we need this condition there as well (I'm not totally sure when each component fires, but I am still seeing rate changed currently 0x polygon -> usdc on polygon)